interop/src/main.rs

```diff
@@ -176,6 +182,50 @@ fn run(log: Logger, options: Opt) -> Result<()> {
         }
     }
 
+    let mut h3 = false;
+    let result = runtime.block_on(
+        endpoint
+            .connect_with(&client_config, &remote, &options.host)?
```

Comment:
```suggestion
            .connect_with(&client_config, &remote, host)?
```
Looks like you missed this in a rebase.

Reply:
fixed in 4ee65b9